benchmark/run-benchmark.js

```diff
@@ -18,8 +18,8 @@ box.min.set( 1, 1, 1 );
 const intersectGeometry = new THREE.TorusBufferGeometry( 5, 5, 100, 50 );
 const geomMat = new THREE.Matrix4().compose( new THREE.Vector3(), new THREE.Quaternion(), new THREE.Vector3( 0.1, 0.1, 0.1 ) );
 
-const target1 = new THREE.Vector3();
-const target2 = new THREE.Vector3();
+const target1 = undefined;
+const target2 = undefined;
```

Comment:
Lets set these to empty objects {} so the objects will be reused and to reduce the impact of garbage collection on the benchmark.